teams/release/charter.md

```diff
@@ -38,10 +36,14 @@ Notably excluded: release cadence, delaying a stable release. (Likely those are
 
 ### [Consulted for input](../../common/darci.md#consulted-for-input)
 
-* Regression triage (shared with T-compiler)
+* Regression triage (shared with teams responsible for each part of the codebase)
```

Comment:
I think it'd actually be good to add a note that we expect (I believe) those teams to be responsible for fixing the regressions.

A more contentious question is who determines acceptable breakage; I think right now that's also those teams, and they should at minimum inform and ideally consult us.